tanner/dbutils.py

```diff
@@ -0,0 +1,100 @@
+import asyncio
+
+
+class CreateTables:
```

Comment:
please differentiate between class and function. I prefer to have a class as a Noun entity which consists of actions: 
e.g. 
```
class DBUtils <Noun>:
<...>
    def create_tables <action (verb)>:
        <...>

```